cognite/v05/depthseries.py

```diff
@@ -0,0 +1,316 @@
+# -*- coding: utf-8 -*-
+"""Depthseries Module
+
+This module mirrors the Timeseries API, but handles pairs of timeseries to emulate depth(or other indexed) data series. It allows you to fetch data from the api and output it in various formats.
+
+https://doc.cognitedata.com/0.5/#Cognite-API-Time-series
+"""
+
+import sys
+
+from typing import List
+import copy
+import itertools
+
+import cognite._utils as _utils
+import cognite.config as config
+
+from cognite.v05 import timeseries
+from cognite.v05.dto import  LatestDatapointResponse, \
+    DatapointDepth, \
+    TimeSeries, TimeSeriesResponse, Datapoint
+
+
+
+
+
+def post_datapoints(name, depthdatapoints: List[DatapointDepth], **kwargs):
+    '''Insert a list of datapoints.
+
+    Args:
+        name (str):       Name of timeseries to insert to.
+
+        datapoints (list[v05.data_objects.Datapoint): List of datapoint data transfer objects to insert.
+
+    Keyword Args:
+        api_key (str): Your api-key.
+
+        project (str): Project name.
+
+    Returns:
+        An empty response.
+    '''
+    name = name.replace('/', '%2F')
```

Comment:
just minor nitpick: you should use urllib.parse.quoteplus to url encode these names